### PR TITLE
Remove moment.js dependencies

### PR DESCRIPTION
**Pull request by jakub-wojciechowski**
Following the discussion with @frangio in PR #353

As `testrpc` uses plain time arithmetic for time manipulation, using moment.js which is based on the implementation of full calendar logic brings a potential risk of indeterministic errors in tests.

### CODE REVIEW

File: test/helpers/increaseTime.js [hunk 1]
```diff
@@ -43,5 +43,7 @@ export const duration = {
   minutes: function(val) { return val * this.seconds(60) },
   hours:   function(val) { return val * this.minutes(60) },
   days:    function(val) { return val * this.hours(24) },
-  weeks:   function(val) { return val * this.days(7) }
+  weeks:   function(val) { return val * this.days(7) },
+  months:  function(val) { return val * this.days(30)},
```
Comment (frangio) [inline test/helpers/increaseTime.js]: I'm thinking we should only provide time units matching [those of Solidity](https://solidity.readthedocs.io/en/develop/units-and-global-variables.html#time-units). `months` is not included and I see you haven't used it anywhere either. What do you think about removing it?

File: test/TokenTimelock.js [hunk 2]
```diff
@@ -29,26 +28,26 @@ contract('TokenTimelock', function ([_, owner, beneficiary]) {
   })
 
   it('cannot be released just before time limit', async function () {
-    await increaseTime(moment.duration(0.99, 'year').asSeconds())
+    await increaseTimeTo(this.releaseTime - duration.seconds(3))
```
Comment (frangio) [inline test/TokenTimelock.js]: I'm surprised this works. Given the lack of precision of `increaseTimeTo` that we've discussed, I would've thought 3 seconds was not enough to make up for it. Did you try other values?

File: test/helpers/increaseTime.js (hunk 1 above)
Comment (jakub-wojciechowski) [inline test/helpers/increaseTime.js]: Yes, it makes a total sense to mimic the solidity constants. Thanks for pointing that out!

**Comment by jakub-wojciechowski** [comment]
@frangio I've included all of the latest changes and generated a new yarn.lock. I suppose the inconsistency is caused by us using different versions of yarn. Can you please tell me yours (mine is 0.24.5) so I can double check if that's the case before committing updates.

File: test/TokenTimelock.js (hunk 2 above)
Comment (jakub-wojciechowski) [inline test/TokenTimelock.js]: As `testrpc` is run on localhost the lag is usually up to 1s. Due to time rounding,  subtracting a single second is not enough to have a robust behaviour but 3s seems to be safe enough. 

Obviously, the situation may change if someone decides to run tests across `testrpc` instance connected. But, to be honest, I don't expect it to be a common practice.

**Comment by frangio** [comment]
yarn version 0.28.4

You should remove your `node_modules` folder and run `yarn install` using the lockfile that is in the repo. Removing `moment` would then result in very small changes to `yarn.lock`.

**Comment by jakub-wojciechowski** [comment]
Switching to version 0.28.4 helped.
I still needed to manually switch positions of two versions of the same dependency that were next to each other (I guess yarn is not 100% deterministic) but apart from moving one paragraph to the top of another, the rest was identical.

**Comment by frangio** [comment]
Excellent. Thanks @jakub-wojciechowski!

**Pull request merged**